docs/papers/utils/minipaper_cracking_symmetry.py

```
@@ -0,0 +1,400 @@
+    b_model = LinearRegression().fit(A, b)
+
+    print(f"Best fit: c ≈ {c_model.coef_[0]:.4f}·|A| + {c_model.intercept_:.4f}")
+    print(f"Best fit: b ≈ {b_model.coef_[0]:.4f}·|A| + {b_model.intercept_:.4f}")
+
+    # Plot
+    plt.figure(figsize=(10, 4))
+
+    plt.subplot(1, 2, 1)
+    plt.scatter(a_mag, c, s=5, alpha=0.3, label='samples')
+    plt.plot(a_mag, c_model.predict(A), 'r-', label='best fit')
+    plt.xlabel('|A|'); plt.ylabel('c')
```

Ruff (0.11.9)

Multiple statements on one line (semicolon)

(E702)